spec/models/student_spec.rb

```diff
@@ -755,4 +755,11 @@
       it { is_expected.to have_abilities(:read, Transcript) }
     end
   end
+
+  describe '.pull_info_from_crm', :vcr do
+    it 'returns name & Epicenter course id for close_io lead given email' do
+      course = FactoryGirl.create(:course, description: "* Placement Test")
+      expect(Student.pull_info_from_crm("example@example.com")).to eq({name: "TEST TEST", course_id: course.id})
+    end
+  end
 end
```

Comment:
also test the error conditions: if an email isn't found, if the email isn't unique, if the course isn't assigned. ideally, write tests for each scenario first, then only write as much code as takes to make the test pass. if you write code that isn't necessary to make the test pass, then you need another test.